cratedb_toolkit/cfr/cli.py

```diff
@@ -46,8 +46,20 @@ def cli(ctx: click.Context, cratedb_sqlalchemy_url: str, verbose: bool, debug: b
 def sys_export(ctx: click.Context, target: str):
     cratedb_sqlalchemy_url = ctx.meta["cratedb_sqlalchemy_url"]
     try:
-        stc = SystemTableExporter(dburi=cratedb_sqlalchemy_url, target=path_from_url(target))
+        target_path = path_from_url(target)
+        stc = SystemTableExporter(dburi=cratedb_sqlalchemy_url, target=target_path)
+
+        archive = None
+        if target_path.name.endswith(".tgz") or target_path.name.endswith(".tar.gz"):
+            archive = Archive(stc)
+
         path = stc.save()
+
+        if archive is not None:
+            path = archive.make_tarfile()
+            archive.close()
+            logger.info(f"Created archive file {target}")
```

Comment:
I like that approach very much, perfect [DWIM](https://en.wikipedia.org/wiki/DWIM).